scripts/sdk/examples/placeLimitOrder.ts

```diff
@@ -1,71 +1,88 @@
+    client.addBalanceManager("MANAGER_1", "0x0c34e41694c5347c7a45978d161b5d6b543bec80702fee6e002118f333dbdfaf");
 
-    // This will split 1 DEEP (1000000) from the deep object ID that was set in init()
-    // and deposit it into the balance manager.
-    await dbClient.depositIntoManager(managerKey, 1, CoinKey.DEEP);
+    return client;
+}
 
-    await dbClient.placeLimitOrder({
-        poolKey: PoolKey.DBWETH_DBUSDC,
-        managerKey: 'MANAGER_1',
+// Example of creating a client, initializing all variables, and placing a buy order of 1 DEEP for price of 2 SUI
+export const placeLimitOrderClient = async () => {
+    const client = await init();
+    const managerKey = "MANAGER_1";
+
+    const txb = new TransactionBlock();
+    await client.depositIntoManager(managerKey, 2, "SUI", txb);
+    await client.placeLimitOrder({
+        poolKey: "DEEP_SUI",
+        managerKey: managerKey,
         clientOrderId: 888,
         price: 2,
         quantity: 1,
         isBid: true,
         // orderType is default: no restriction
         // selfMatchingOption is default: allow self matching
         // payWithDeep is default: true
-    })
+    }, txb);
 }
 
-// Here, instead of using multiple function calls that the client provides, we will construct a
-// custom PTB to do all of the above in one single transaction.
-export const placeLimitOrderPTB = async () => {
-    const pk = process.env.PRIVATE_KEY as string;
-    const managerKey = "MANAGER_1";
-    const managerId = "0x0c34e41694c5347c7a45978d161b5d6b543bec80702fee6e002118f333dbdfaf";
-    const dbClient = new DeepBookClient("testnet", pk);
-    await dbClient.init(true);
-    dbClient.addBalanceManager(
-        managerKey,
-        managerId,
+export const placeLimitOrderBorrowDeep = async () => {
+    const client = await init();
+    const txb = new TransactionBlock();
+    const borrowAmount = 10;
+    const [deepCoin, flashLoan] = await client.borrowBaseAsset(
+        "DEEP_SUI",
+        borrowAmount,
+        txb
     )
-    const balanceManager = dbClient.getBalanceManager(managerKey);
 
-    let txb = new TransactionBlock();
+    // Execute trade using borrowed DEEP
+    const [baseOut, quoteOut, deepOut] = await client.swapExactQuoteForBase({
+        poolKey: "SUI_DBUSDC",
+        amount: 0.5,
+        deepAmount: 10,
+        deepCoin: deepCoin,
+    }, txb);
 
-    const deepCoin = dbClient.getConfig().getCoin(CoinKey.DEEP);
-    const deepSuiPool = dbClient.getConfig().getPool(PoolKey.DEEP_SUI);
+    txb.transferObjects([baseOut, quoteOut, deepOut], client.getActiveAddress());
 
-    depositIntoManager(balanceManager.address, 1, deepCoin, txb);
-    // explicitely set order type, self matching, and payWithDeep
-    placeLimitOrder(deepSuiPool, balanceManager, 12345, 1, 1, false, MAX_TIMESTAMP, 0, 0, true, txb);
+    // Execute second trade to get back DEEP for repayment
+    const [baseOut2, quoteOut2, deepOut2] = await client.swapExactQuoteForBase({
```

Comment:
how do we know that baseOut2 in this case is exactly 10 DEEP? Don't we have to split 10 DEEP from it then return the flashloan?

Reply:
So def up for discussion, the return loan function will split the coin automatically if it's above the loan threshold it'll send the remaining back to the user